### PR TITLE
fix: add support for bazel BuildSteps generation

### PR DESCRIPTION
**Pull request by danielmoy-google**
Uses Cody's new `gcr.io/kythe-public/bazel-extractor`.  Some small diffs on the README and variable names there.  Also adopt /workspace/output for the rest of the extractors because that reads a lot nicer than /workspace/out that I was using before.

### CODE REVIEW

File: kythe/go/extractors/gcp/config/gradle.go [hunk 1]
```diff
@@ -21,7 +21,6 @@ import (
 	"path/filepath"
 
 	"kythe.io/kythe/go/extractors/constants"
-
```
Comment (schroederc) [inline kythe/go/extractors/gcp/config/gradle.go]: Restore blank line

File: kythe/go/extractors/gcp/config/maven.go [hunk 2]
```diff
@@ -21,7 +21,6 @@ import (
 	"path/filepath"
 
 	"kythe.io/kythe/go/extractors/constants"
-
```
Comment (schroederc) [inline kythe/go/extractors/gcp/config/maven.go]: Restore blank line

Comment (danielmoy-google) [reply kythe/go/extractors/gcp/config/maven.go]: :+1: 

(bazel.go too)

File: kythe/go/extractors/gcp/config/gradle.go (hunk 1 above)
Comment (danielmoy-google) [reply kythe/go/extractors/gcp/config/gradle.go]: :+1:

**Pull request merged**